packages/hint/tests/lib/cli/analyze.ts

```diff
@@ -195,15 +193,18 @@ test.serial('If config file does not exist, it should use `web-recommended` as d
     sandbox.stub(config.Configuration, 'validateHintsConfig').returns(validateHintsConfigResult);
 
     sandbox.stub(askQuestion, 'default').resolves(false);
-    await t.notThrows(analyze(actions));
+
+    t.plan(2);
```

Comment:
Do we need `plan` here?
https://github.com/avajs/ava/blob/master/docs/recipes/when-to-use-plan.md#promises-that-are-expected-to-resolve

Reply:
nope, we don't need it here.